### ISSUE TITLE
Unable to deploy syndesis from master

### ISSUE THREAD

**Issue by avano** [syndesisio/syndesis#5442]
## This is a...

<pre><code>
[ ] Feature request
[X] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description
Using `./tools/bin/syndesis install --project syndesis -y --test-support`:

{"level":"info","ts":1558520033.689927,"logger":"action","msg":"Installing Syndesis resource","type":"install","name":"app"}
{"level":"error","ts":1558520033.9660332,"logger":"controller","msg":"Error reconciling","action":"*action.installAction","phase":"Installing","error":"failed to decode json data with gvk(apps.openshift.io/v1, Kind=DeploymentConfig): v1.DeploymentConfig.Spec: v1.DeploymentConfigSpec.Replicas: readUint32: unexpected character: \ufffd, error found in #10 byte of ...|eplicas\":\"0\",\"select|..., bigger context ...|ure\"},\"name\":\"komodo-server\"},\"spec\":{\"replicas\":\"0\",\"selector\":{\"app\":\"syndesis\",\"syndesis.io/app\":|...","stacktrace":"github.com/syndesisio/syndesis/install/operator/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/github.com/go-logr/zapr/zapr.go:128\ngithub.com/syndesisio/syndesis/install/operator/pkg/controller/syndesis.(*ReconcileSyndesis).Reconcile\n\t/go/src/github.com/syndesisio/syndesis/install/operator/pkg/controller/syndesis/syndesis_controller.go:120\ngithub.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:213\ngithub.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:158\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:134\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.Until\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88"}
{"level":"error","ts":1558520033.9661355,"logger":"kubebuilder.controller","msg":"Reconciler error","controller":"syndesis-controller","request":"syndesis/app","error":"failed to decode json data with gvk(apps.openshift.io/v1, Kind=DeploymentConfig): v1.DeploymentConfig.Spec: v1.DeploymentConfigSpec.Replicas: readUint32: unexpected character: \ufffd, error found in #10 byte of ...|eplicas\":\"0\",\"select|..., bigger context ...|ure\"},\"name\":\"komodo-server\"},\"spec\":{\"replicas\":\"0\",\"selector\":{\"app\":\"syndesis\",\"syndesis.io/app\":|...","stacktrace":"github.com/syndesisio/syndesis/install/operator/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/github.com/go-logr/zapr/zapr.go:128\ngithub.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:215\ngithub.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:158\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:134\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.Until\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88"}

**Comment by heiko-braun** [comment]
@zregvart Isn't this the same thing you discussed on the recent PR?

```
unexpected character: \ufffd
```

**Comment by zregvart** [comment]
Yes, I'll separate a fix for this in a PR so we can fix the template right away.

**Comment by avano** [comment]
@zregvart same steps latest master:

```
{"level":"error","ts":1558616367.7950253,"logger":"kubebuilder.controller","msg":"Reconciler error","controller":"syndesis-controller","request":"syndesis/example","error":"open emplate: no such file or directory","stacktrace":"github.com/syndesisio/syndesis/install/operator/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/github.com/go-logr/zapr/zapr.go:128\ngithub.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:217\ngithub.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:158\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:134\ngithub.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait.Until\n\t/go/src/github.com/syndesisio/syndesis/install/operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88"}

```

```
"error":"open emplate: no such file or directory"
```
:thinking:

**Comment by zregvart** [comment]
@avano this is using `./tools/bin/syndesis install --project syndesis -y --test-support` right?

**Comment by zregvart** [comment]
Could be the `-template` vs `--template` change introduced with #5446

**Comment by zregvart** [comment]
I just did a:
```
$ minishift start
$ oc login -u admin -p supersecret
$ ./tools/bin/syndesis install --project syndesis -y --test-support
```

And got a running Syndesis installation. The operator pod log only had a different error logged (#5458) though that didn't prevent the app from starting and the status on the Syndesis custom resource becoming:

```yaml
  status:
    phase: Installed
    version: latest
```

@avano can you share how you got it to fail?

**Comment by avano** [comment]
right, ./tools/bin/syndesis install --project syndesis -y --test-support . Im not at the pc at the moment, I will retry in ~1.5hrs

**Comment by phantomjinx** [comment]
@zregvart 
I've just repeated the error @avano has:
1. Updated master branch to latest
2. `syndesis install --dev --watch --local --force -y`

The --local is officially using my syndesis src.

If that helps.

**Comment by avano** [comment]
@zregvart I thought that our jenkins runs the command I posted, but it also uses `--local` as @phantomjinx mentioned.

I can confirm that it works with `./tools/bin/syndesis install --project syndesis -y --test-support` and fails with `./tools/bin/syndesis install --project syndesis -y --test-support --local`


The culprit will probably be that the configmap `syndesis-template` that is created with `--local` has:

```
Command:
      syndesis-operator
      -template
      /conf/syndesis-template.yml
```

**Comment by zregvart** [comment]
Fixed via #5461
